back/engines/commercial/idea_custom_fields/lib/tasks/add_custom_fields.rake

```diff
@@ -0,0 +1,25 @@
+namespace :setup_and_support do
```

Comment:
We won't get yml files in the right format, which will make support work very high effort. Can we rethink this?